admin-tools/lambda/src/main/scala/com/gu/mediaservice/BatchIndexLocalHandler.scala

```diff
@@ -0,0 +1,36 @@
+package com.gu.mediaservice
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object BatchIndexLocalHandler extends App {
+
+  lazy val awsCredentials = new ProfileCredentialsProvider("media-service")
+
+  private val cfg = BatchIndexHandlerConfig(
+    apiKey = "dev-",
+    domainRoot = "local.dev-gutools.co.uk",
+    batchIndexBucket = "media-service-admin-tools-dev",
+    kinesisStreamName = "media-service-DEV-ThrallMessageQueue-1N0T2UXYNUIC9",
+    dynamoTableName = "grid-images-to-reingest-DEV",
+    batchSize = 1,
+    kinesisEndpoint = Some("http://localhost:4568"),
+    awsCreds = Some(awsCredentials)
+  )
+
+  private val batchIndex = BatchIndexHandler(cfg)
+
+  // ids that exists on your local grid
+  val mediaIds = List(
+    "011db3facffd4ef3d3e07eabbd3c07bb41cbf819",
```

Comment:
Can this be pushed to input rather than hard coded?

Reply:
done